docs/builders/iso.mdx

```diff
@@ -82,6 +82,9 @@ in the image's Cloud-Init settings for provisioning.
 
 ### Optional:
 
+- `iso_download_pve` (bool) - Download the specified `iso_url` directly to
+  the PVE node. Defaults to `false`
```

Comment:
If you don't mind me chiming in for this, I think the comment might be slightly confusing with respects to the `to the PVE node`, my understanding is that Packer delegates downloading the ISO to the server, so in this case the ISO would be downloaded directly from the server, and Packer only gives the URI to get the ISO from.

I also think that users unfamiliar with the current behaviour may have a hard time deciding if this option is for them based on this information only, I think we should explain a bit more why that option exists and what it implies compared to the default behaviour.

I'll suggest a rephrasing, feel free to adapt it however you see fit.
```
- `iso_download_pve` (bool) - Download the specified `iso_url` directly from
  the PVE node. Defaults to `false`.
  By default Packer downloads the ISO and uploads it in a second step, this
  option lets Proxmox handle downloading the ISO directly from the server.
```

Reply:
Thanks, as a non-native speaker I sometimes have a hard time to decide between _downloading from/to/onto_.
I guess your version makes it all more clear.

Reply:
Kein problem, happens to all of us!
Thanks for making the edit, I'll merge this PR as-is.